blocks/init/src/blocks/components/menu-drawer/menu-drawer-style.scss

```diff
@@ -0,0 +1,71 @@
+$menu-drawer: (
+  width: 256px,
+  colors: (
+    background: #FFFFFF,
+    link: $base-text-color,
+    link-hover: $base-primary-color
+  ),
+  navigation: (
+    font-size: 24px,
+    font-family: $base-font-family
+  ),
+  link-offset: 24px,
+  vertical-padding: 26px,
+  padding-navigation: 16px,
+);
+
+body {
```

Comment:
why wrap it in body

Reply:
Same thing as https://github.com/infinum/eightshift-frontend-libs/pull/83#discussion_r382567406, because we're using a class on body to see if menu is open or not. Alternative is (same as in the previous case) that we have 2 top level selectors in this file.

Reply:
🆙